compiler-core/src/type_/tests/warnings.rs

```diff
@@ -2671,3 +2671,27 @@ pub fn go() {
 "#
     );
 }
+
+#[test]
+fn javascript_unsafe_int_with_external_implementation() {
+    assert_js_no_warnings!(
+        r#"
+@external(javascript, "./test.mjs", "go")
+pub fn go() -> Int {
+  9_007_199_254_740_992
+}
+"#
+    );
+}
```

Comment:
Could we have a test that this code emits a warning please

```gleam
pub fn main() {
  9_007_199_254_740_992 + helper()
}

@external(javascript, "a", "b")
fn helper() -> Int
```

Reply:
Added extra test for this